docs/using.md

```diff
@@ -5,6 +5,7 @@
 * [How do I make Resources?](#creating-resources)
 * [How do I run a Pipeline?](#running-a-task)
 * [How do I run a Task on its own?](#running-a-task)
+* [How do I troubleshoot a PipelineRun](#troubleshooting)
```

Comment:
plz add a `?` to be consistent with the other links

Reply:
ugh im really sorry to block this going in on just the `?` :( feel free to merge without the `?` if you want